packages/e2e-test-utils-playwright/src/page/site-editor.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import type { PageUtils } from './index';
+
+const VISUAL_EDITOR_SELECTOR = '.edit-site-visual-editor iframe';
```

Comment:
I wonder if we could select it by its `title` or `name`?

Reply:
Name didn't seem to work, so I did a standard attribute selector for the title 👍